deployment/docker_compose/docker-compose.search-testing.yml

```diff
@@ -0,0 +1,406 @@
+version: '3'
```

Comment:
I'm quite against having yet another docker compose yaml. Every time we add a new env based config we have to currently copy it to 3 docker compose yamls, kubernetes deployment, helm deployment. Let's see if we can just use the existing yamls